renovate.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Codacy found an issue: [Use the latest vue-eslint-parser. See also https://eslint.vuejs.org/user-guide/#what-is-the-use-the-latest-vue-eslint-parser-error.](https://app.codacy.com/manual/liannoi/team-project/pullRequest?prid=6058387)